### ISSUE TITLE
Drug entry text field missing label

### ISSUE THREAD

**Issue by grugnog** [CivicActions/nebula#70]
This has placeholder text, but not all assistive technology will make that text accessible to users. We should add a label associated with the field that indicates what should be entered (e.g. http://webaim.org/techniques/forms/controls#input).

We have a couple of options:
- We could visually hide the label - this will not affect the design, but will still provide the text for screen readers. For working style see .visually-hidden class on http://cgit.drupalcode.org/drupal/tree/core/modules/system/css/system.module.css#n321 (note that  display:none won't work for this).
- We could add a visible label to the left of the field, and make the placeholder more of an "example" drug name, which I think is a more typical usage.

I think the latter is slightly preferable from an accessibility perspective (since there is less cognitive demand due to the placeholder label disappearing once you click on the text box), however given that it is a very simple single field form I don't think there is a major difference.

**Comment by rakanowicz** [comment]
I like the hidden label and adding an "example" drug.
